pkg/internal/checkup/checkup.go

```diff
@@ -210,3 +206,14 @@ func randomizeName(prefix string) string {
 func ObjectFullName(namespace, name string) string {
 	return fmt.Sprintf("%s/%s", namespace, name)
 }
+
+func CloudInit(username, password string) string {
```

Comment:
In the DPDK checkup there is s [unit test](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/614adcb89fab7518d8c8a513c61dab99e1a8fd2b/pkg/internal/checkup/vmi_test.go#L92) for this function (this is the reason its exported).
Please consider adding it.

Reply:
I think it got lost on the conflict resolution and I missed it. will fix

Reply:
DONE